identity/client/src/pages/users/List.tsx

```diff
@@ -71,7 +71,7 @@ const List: FC = () => {
     <Page>
       <EntityList
         title={t("Users")}
-        data={userSearchResults!.items}
+        data={userSearchResults == null ? [] : userSearchResults.items}
```

Comment:
Is this change needed for the license call to work?

Reply:
It's not directly related as it fixes another bug I discovered during the test, but without this the page can't be rendered and just shows error page.